homeassistant/components/dlna_dms/const.py

```diff
@@ -42,40 +42,40 @@
 
 # Map UPnP object class to media_player media class
 MEDIA_CLASS_MAP: Mapping[str, str] = {
```

Comment:
The type should now be `Mapping[str, MediaClass]`.

Reply:
There are two `MediaType` strings in there, so it would have to be `Mapping[str, MediaClass | MediaType]`

Or maybe those two MediaType should be dropped/amended?

Reply:
Ah, I missed those. Given `MEDIA_CLASS_MAP` is only used for setting `BrowseMediaSource.media_class`, we should stick with only `MediaClass` values. Reading the [UPnP AV:ContentDirectory v4 spec](https://openconnectivity.org/wp-content/uploads/2015/11/UPnP-av-ContentDirectory-v4-Service-20101231.pdf) (page 259), those are most similar to "object.container.channelGroup", so `MediaClass.CHANNEL` would suffice.

As a side note, the bad use of `MEDIA_TYPE_CHANNELS` was a hold-over from when the media class was much more loosely defined. I'm happy to see it get stricter.

Reply:
I opened https://github.com/home-assistant/core/pull/78458 to adjust it